### DISCUSSION TITLE
When using Opentelementry auto instrumentation java agent with glowroot agent, trace_id and span_id not generating for java executors framework.

### DISCUSSION THREAD

**Discussion by bhaveshmodiatekaplus** [open-telemetry/opentelemetry-java-instrumentation#6281] [Q&A]
Hi @trask  / All , 

	I want to use glowroot for APM and Opentelementry auto instrumentation for injecting trace_id and span_id in logs.
	I am using the below command to start Jetty and configure javaagent.
	
    java -javaagent:./glowroot/opentelemetry-javaagent.jar  -Dotel.traces.exporter=none -Dotel.metrics.exporter=none -Dotel.service.name=PlatformService   -javaagent:./glowroot/glowroot.jar  -jar start.jar
	
	For normal senarios , traceid and spanid is generated. but when it comes to java thread executors traceid and spaceid are coming blank. 
	
	When I remove glowroot agent from startup script, it works fine we can view traceid and spanid for java thread executors
	
	Can anyone have faced this kind of issues , please have a look and suggest me better approach.
	
	Thanks in Advance

**Comment by trask** [comment]
hi @bhaveshmodiatekaplus! this is not something we support in OpenTelemetry Javaagent, see discussion at #1534 for more background
